manifests/b/Blitz/Blitz/2.1.100/Blitz.Blitz.installer.yaml

```diff
@@ -0,0 +1,19 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Blitz.Blitz
+PackageVersion: 2.1.100
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+Scope: user
+InstallModes:
+- silent
+UpgradeBehavior: install
+Installers:
+- Architecture: x86
```

Comment:
```suggestion
- Architecture: x64
```

According to the package homepage, this is a 64-bit installer file.

![image](https://github.com/microsoft/winget-pkgs/assets/49941141/4ec1e5da-ce68-48ac-a72e-f0e15e10504e)